src/config/voice.c

```diff
@@ -39,44 +40,43 @@ static const char* getfield(char* value, int num) {
 
 static int ini_handler(void* user, const char* section, const char* name, const char* value)
 {
-    (void) user;
+    u16 *req_id = user;
     char tmp[100];
     strlcpy(tmp, value, 100);
     int duration = atoi(getfield(tmp,2));
-#if HAS_MUSIC_CONFIG
-    char label[MAX_VOICE_LABEL];
-    strlcpy(label,getfield(tmp, 1), MAX_VOICE_LABEL);
-#endif
+    u16 id = atoi(name);
```

Comment:
this is the only place in all of deviation that strtok() is called.  Is there an alternative that wouldn't require including it (or the strcpy that goes with modifying the string)?

Reply:
removed stroke usage and used temp string